.yarn/versions/70b508e3.yml

```diff
@@ -0,0 +1,24 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/plugin-http": patch
```

Comment:
This is a new feature, no?

Reply:
There's a reasonable case that it's a compatibility bugfix. It could be seen as a feature as well, but I don't think it's matches the typical threshold for what's worth waiting for a minor release vs a patch one.

Reply:
> waiting for a minor release

Don't wait, make releases whenever we feel like.